### PR TITLE
[ui-storagebrowser] adds and reads configs from .ini files

### PR DESCRIPTION
**Pull request by ramprasadagarwal**
## What changes were proposed in this pull request?

- Reads the config from .ini file and uses it for file browser components

## How was this patch tested?

- Updated unit tests

### CODE REVIEW

File: desktop/core/src/desktop/api2.py [hunk 1]
```diff
@@ -128,6 +129,10 @@ def get_config(request):
   config['hue_config']['enable_chunked_file_uploader'] = ENABLE_CHUNKED_FILE_UPLOADER.get()
   config['hue_config']['enable_task_server'] = TASK_SERVER_V2.ENABLED.get()
   config['hue_config']['restrict_file_extensions'] = RESTRICT_FILE_EXTENSIONS.get()
+  config['hue_config']['concurrent_max_connection'] = CONCURRENT_MAX_CONNECTIONS.get()
+  config['hue_config']['file_upload_chunk_size'] = FILE_UPLOAD_CHUNK_SIZE.get()
+  config['hue_config']['enable_file_download_button'] = SHOW_DOWNLOAD_BUTTON.get()
```
Comment (Harshg999) [inline desktop/core/src/desktop/api2.py:134]: These configs are already present in `desktop/core/src/desktop/templates/global_js_constants.mako`

Comment (ramprasadagarwal) [reply desktop/core/src/desktop/api2.py:134]: Nice catch, removed it from global file.

**Review by JohanAhlen — CHANGES_REQUESTED** [review]
The revert of the unsafe-inline shouldn't be part of this PR right?

File: desktop/conf.dist/hue.ini [hunk 2]
```diff
@@ -137,9 +137,6 @@ http_500_debug_mode=false
 # X-Content-Type-Options: nosniff This is a HTTP response header feature that helps prevent attacks based on MIME-type confusion.
 ## secure_content_security_policy="script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googletagmanager.com *.doubleclick.net data:;img-src 'self' *.doubleclick.net http://*.tile.osm.org *.tile.osm.org *.gstatic.com data:;style-src 'self' 'unsafe-inline' fonts.googleapis.com;connect-src 'self' *.google-analytics.com;frame-src *;child-src 'self' data: *.vimeo.com;object-src 'none'"
 
-# Enable nonce attribute to remove unsafe-inline and auto remove unsafe-inline from csp 
-## csp_nonce=true
-
```
[the remark this answers is not in the document]
Comment (ramprasadagarwal) [reply desktop/conf.dist/hue.ini:142]: Yes, By mistake it got pushed. Thanks for flagging.

File: desktop/core/src/desktop/api2.py (hunk 1 above)
Comment (ramprasadagarwal) [reply desktop/core/src/desktop/api2.py]: Make sense, I have moved it. Thanks

**Review by JohanAhlen — APPROVED** [review]
Nice and clean!

**Pull request merged**